plugin/session_buffer.py

```diff
@@ -160,6 +166,10 @@ def session_views(self) -> WeakSet[SessionViewProtocol]:
     def last_synced_version(self) -> int:
         return self._last_synced_version
 
+    @property
+    def supported_commands(self) -> set[str]:
+        return self._supported_commands
```

Comment:
I would think that the purpose of creating getters like this one is to make it explicit that this is a public property. In this case this is only used within this class so this seems redundant.